test/foundry/fixtures/CustomERC721Fixture.t.sol

```diff
@@ -200,9 +202,6 @@ contract CustomERC721Fixture is Test {
 
     // Connect the drop implementation to the drop proxy address
     customErc721 = CustomERC721(payable(newDropAddress));
-
-    vm.prank(DEFAULT_OWNER_ADDRESS);
-    return customErc721.initLazyMint();
```

Comment:
should this be reenabled?

Reply:
the `initLazyMint` function sync the `CustomERC721` tokenId with the lazyMint one

I have renamed it to `syncLazyMint` 👍🏻